uiowa/uiowa
# Update Area of Study and Transfer tip long text fields to `filtered_html`.

Resolves #3419 

# How to test

- `blt ds --site admissions.uiowa.edu` and watch the output.
- Spot check some items from the list below to make sure that the format was updated correctly. 
- Check older revisions, if they exist, of items in the list below to make sure that  the format was updated correctly across all revisions.
- `blt ds` and confirm no additional output for update hook.

## Spot check list:
* https://admissions.local.drupal.uiowa.edu/node/2736
* https://admissions.local.drupal.uiowa.edu/node/2776
* https://admissions.local.drupal.uiowa.edu/node/2956
* https://admissions.local.drupal.uiowa.edu/node/3026
* https://admissions.local.drupal.uiowa.edu/node/3106
* https://admissions.local.drupal.uiowa.edu/node/3111
* https://admissions.local.drupal.uiowa.edu/node/3126
* https://admissions.local.drupal.uiowa.edu/node/3236
* https://admissions.local.drupal.uiowa.edu/node/3261
* https://admissions.local.drupal.uiowa.edu/node/3301
* https://admissions.local.drupal.uiowa.edu/node/3391
* https://admissions.local.drupal.uiowa.edu/node/3416
* https://admissions.local.drupal.uiowa.edu/node/3421
* https://admissions.local.drupal.uiowa.edu/node/3431
* https://admissions.local.drupal.uiowa.edu/node/3441
* https://admissions.local.drupal.uiowa.edu/node/3636
* https://admissions.local.drupal.uiowa.edu/node/3661
* https://admissions.local.drupal.uiowa.edu/node/3666
* https://admissions.local.drupal.uiowa.edu/node/3676
* https://admissions.local.drupal.uiowa.edu/node/3811
